.eslintrc.js

```diff
@@ -6,15 +6,18 @@ module.exports = {
     project: ['./tsconfig.json'],
   },
   plugins: [
-    'eslint-plugin-tsdoc',
-    'eslint-plugin-import',
-    'eslint-plugin-unused-imports'
+    'tsdoc',
+    'import',
+    'jest',
```

Comment:
Since we already have an `.eslintrc.js` specifically for the test folder I would put the plugin there instead.

Reply:
Slight issue is that I might be overriding the `plugins` array instead of appending to it.

Reply:
I just tried it out and all the other rules still seem to be working when I move the jest stuff to the nested lint file. Or is there something else that breaks?

Reply:
Indeed, saw the same.